conf/radon.default.json

```diff
@@ -2,7 +2,8 @@
         "proxy": {
                 "endpoint": ":3308",
                 "meta-dir": "bin/radon-meta",
-                "peer-address": ":8080"
+                "peer-address": ":8080",
+                "long-query-time": "5"
```

Comment:
There is no need add long-query-time to the radon.default.json, just keep in the default is ok.